plugins/auth-backend/src/lib/oauth/OAuthAdapter.ts

```diff
@@ -165,9 +165,11 @@ export class OAuthAdapter implements AuthProviderRouteHandlers {
     }
 
     if (!this.handlers.refresh || this.options.disableRefresh) {
-      res.send(
-        `Refresh token not supported for provider: ${this.options.providerId}`,
-      );
+      res
+        .status(406)
```

Comment:
406 is specific to content negotiation, should probably be 422 or 400 instead. But good eye, to see that the status was missing